st2common/st2common/util/param.py

```diff
@@ -150,8 +152,32 @@ def _render(node, render_context):
     Render the node depending on its type
     '''
     if 'template' in node:
+        complex_type = False
+
+        LOG.debug("Rendering param of type: %s", type(node['template']))
```

Comment:
Do we have access to the param name or similar here? If not the log message seems bit confusing / not so useful since it only contains a type and not any other additional context which could help track it back or similar...